### PR TITLE
save shifted versions of '=', ',', '.' to config

### PR DESCRIPTION
**Pull request by rfomin**
Fixes this issue: [[doomworld]](https://www.doomworld.com/forum/topic/112333-this-is-woof-900-feb-24-2022-updated-winmbf/?do=findComment&comment=2478179)

### CODE REVIEW

File: Source/m_misc.c [hunk 1]
```diff
@@ -2379,7 +2379,20 @@ void M_SaveDefaults (void)
           {
             case input_type_key:
               if (v->value >= 33 && v->value <= 126)
-                fprintf(f, "%c", v->value);
+              {
+                // The '=', ',', and '.' keys originally meant the shifted
+                // versions of those keys, but w/o having to shift them in
+                // the game.
+                char c = v->value;
+                if (c == ',')
```
Comment (fabiangreffrath) [inline Source/m_misc.c:2387]: else if?

**Comment by fabiangreffrath** [comment]
`-` and `_` are no problem?

**Comment by rfomin** [comment]
> `-` and `_` are no problem?

I think not, because you can't type `_` in the menu.

File: Source/m_misc.c [hunk 2]
```diff
@@ -1302,7 +1302,7 @@ default_t defaults[] = {
     NULL, NULL,
     {0}, {UL,UL}, input, ss_keys, wad_no,
     "key to take a screenshot (devparm independent)",
-    input_screenshot, { {input_type_key, '*'} }
+    input_screenshot, { {input_type_key, KEYD_PRTSCR} }
```
Comment (rfomin) [inline Source/m_misc.c:1305]: I don't know what `*` is supposed to mean? Doesn't work in DOSBox either.

Comment (fabiangreffrath) [reply Source/m_misc.c:1305]: That'd be Shift+8 on a US keyboard, strange.

Comment (rfomin) [reply Source/m_misc.c:1305]: Maybe it's a problem with my laptop keyboard, but `*` doesn't work. I think PrtScr is better anyway.

Comment (fabiangreffrath) [reply Source/m_misc.c:1305]: Definitely!

**Review by fabiangreffrath — APPROVED** [review]

**Comment by fabiangreffrath** [comment]
Now I see, this is the pendant to `M_GetKeyString()`.

**Pull request merged**